back/src/Services/SocketManager.ts

```diff
@@ -72,11 +72,17 @@ function emitZoneMessage(subMessage: SubToPusherMessage, socket: ZoneSocket): vo
     socket.write(batchMessage);
 }
 
+interface MeetingData {
+    userId: number;
+    meetingLink: string;
+}
+
 export class SocketManager {
     //private rooms = new Map<string, GameRoom>();
     // List of rooms in process of loading.
     private roomsPromises = new Map<string, PromiseLike<GameRoom>>();
-    private webexMeetings = new Map<string, string>();
+    private webexMeetings = new Map<string, MeetingData>();
```

Comment:
Why do we need to keep any meeting data in our backend? I mean could we get all the needed information from Webex any time we need it?

Reply:
Fair! I thought to store the meeting data in the backend because the events handling users entering and exiting rooms was already there, however if you think it would be a better idea to fetch it from WebEx, we can do that! :)